docs/_docs/quickstart.md

```diff
@@ -4,25 +4,97 @@ title: Quick-start guide
 permalink: /docs/quickstart/
 ---
 
-For the impatient, here's how to get a boilerplate Jekyll site up and running.
+If you already have [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements/)), you can create a new Jekyll site by doing the following:
 
 ```sh
-~ $ gem install jekyll bundler
-~ $ jekyll new myblog
+~ $ gem install jekyll bundler 
+# installs Jekyll and Bundler through RubyGems
+~ $ jekyll new myblog 
+# Creates a new Jekyll site called myblog
 ~ $ cd myblog
-~/myblog $ bundle exec jekyll serve
-# => Now browse to http://localhost:4000
+~/myblog $ bundle exec jekyll serve 
+# Builds the site on the preview server
+# Now browse to http://localhost:4000
 ```
 
-The `jekyll new` command now automatically initiates `bundle install` and installs the dependencies required. To skip this, pass `--skip-bundle` option like so `jekyll new myblog --skip-bundle`.
+The `jekyll new myblog` command installs a new Jekyll site into a subdirectory called `myblog`. The `jekyll new` command automatically initiates `bundle install` to install the dependencies required. (To skip `bundle install`, you can add the following flag: `jekyll new myblog --skip-bundle`.) {% comment %} why would someone want to skip bundle install -- seems unnecessary to even mention it. {% endcomment %}
 
```

Comment:
I don't think a `{% comment %}` tag block is correct.. it is still visible on Github.. 
To answer your query, it was included to give users with latest releases already installed or those who may want to alter their Gemfile before bundling, to simply skip this step..

e.g:

scenario 1:
I install a new Jekyll site and it *automatically* installs all the required versions of gems required by Jekyll.
After an hour or so, if I were to install a new site for another project, I'd use the `skip-bundle` switch as I dont need to fetch any new gems from Rubygems..

scenario 2
I "know" minima is the default theme bundled by Jekyll.. but I'd like to use another theme gem entirely.
So instead of installing unnecessary minima, I `skip-bundle`,  alter my Gemfile accordingly and *then*  run `bundle install`